src/ensembl/src/content/app/entity-viewer/state/gene-view/transcripts/geneViewTranscriptsSlice.ts

```diff
@@ -126,6 +136,14 @@ export const setFilters =
         filters
       })
     );
+
+    entityViewerStorageService.updateGeneViewTranscriptsState({
+      [activeGenomeId]: {
+        [activeEntityId]: {
+          filters
+        }
+      }
+    });
```

Comment:
How do we avoid growing this storage object to unnecessary lengths?

Reply:
That's done here:
https://github.com/Ensembl/ensembl-client/pull/645/files#diff-c08f248640e741c579d69650b5156f8c6391dd4226beb2a57ec4cc25c51af16fR157